test/src/main/java/org/apache/accumulo/test/AuditMessageIT.java

```diff
@@ -112,10 +111,6 @@ private static long findAuditMessage(ArrayList<String> input, String pattern) {
       throw new IOException("Interrupted waiting for data to be flushed to output streams");
     }
 
-    for (MiniAccumuloClusterImpl.LogWriter lw : getCluster().getLogWriters()) {
-      lw.flush();
```

Comment:
I ran 'mvn test' and there were no failures but I will investigate the point you have made.  Thank you.

Reply:
I think `mvn test` only runs unit test and not integration test.   I have the following little script that I use to run individual integration test.

```bash
$ cat ~/bin/runIT 
#!/bin/bash

mvn  -Dit.test="$1" -Dtest=foo  -Dspotbugs.skip -Dcheckstyle.skip -Drat.skip -Daccumulo.skip -Dapilyzer.skip -Dmdep.analyze.skip -Dmodernizer.skip -Dimpsort.skip -Dformatter.skip clean verify
```

With this script when  I do `~/bin/runIT AuditMessageIT` it runs that IT.

Reply:
Really appreciate that information.  Thanks for sharing your trade craft.

Reply:
I ran AuditMessageIT inside the Intellij debugging environment and all the tests pass.  I will run it from the command line like you do tomorrow.

Reply:
> I ran AuditMessageIT inside the Intellij debugging environment

That's perfect.  When running ITs from an IDE its possible they may hang because the native maps are not built.  If this happens then you can run `mvn package -DskipTests` on the command line to build the native maps.  We need to make mini fail in this situation instead of hang.

Reply:
The debug output that AuditMessageIT makes without the logs being flushed is much larger:

-rw-r--r--.  1 jzeiberg dev 1046718 Jan 15 17:42 AuditMessageIT-Accumulo810.txt
drwxr-xr-x. 17 jzeiberg dev    4096 Jan 15 17:43 my_accumulo_fork
drwxr-xr-x.  9 jzeiberg dev     251 Jan 15 17:54 .
-rw-r--r--.  1 jzeiberg dev   85458 Jan 15 17:54 AuditMessageIT-AccumuloMaster.txt